From 3cae331d5253e256f5931a631dd5d4f022d49a85 Mon Sep 17 00:00:00 2001
From: Lu Hui <luhui@sipeed.com>
Date: Tue, 28 Nov 2023 14:45:49 +0800
Subject: [PATCH 4/6] dts: suniv-f1c100s: use a10 compatible usb controller

---
 arch/arm/dts/suniv-f1c100s.dtsi | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/arch/arm/dts/suniv-f1c100s.dtsi b/arch/arm/dts/suniv-f1c100s.dtsi
index 3c61d59ab5..a0218763a3 100644
--- a/arch/arm/dts/suniv-f1c100s.dtsi
+++ b/arch/arm/dts/suniv-f1c100s.dtsi
@@ -134,7 +134,8 @@
 		};
 
 		usb_otg: usb@1c13000 {
-			compatible = "allwinner,suniv-f1c100s-musb";
+			//compatible = "allwinner,suniv-f1c100s-musb";
+			compatible = "allwinner,sun4i-a10-musb";
 			reg = <0x01c13000 0x0400>;
 			clocks = <&ccu CLK_BUS_OTG>;
 			resets = <&ccu RST_BUS_OTG>;
-- 
2.34.1

